src/NzbDrone.Core/Backup/BackupService.cs

```diff
@@ -205,9 +205,10 @@ private void BackupConfigFile()
 
         private void CreateVersionInfo()
         {
-            var builder = new StringBuilder();
+            var tempFile = Path.Combine(_backupTempFolder, "VERSION");
+            var contents = $"v{BuildInfo.Version}-{DateTime.Now:yyyy-MM-dd HH:mm:ss}";
```

Comment:
Is including a different time stamp than the file name necessary?

Reply:
I guess my concerns were two fold.

1. The time stamp here differs from the file name of the backup
2. Was it necessary at all.

Sounds like 2 is a yes, but a consistent time stamp should be used for both the filename and the info in the file.

Reply:
shouldnt the start time of the backup be used for the filename+referred to the file instead of it being generated on each event which then means different times can be seen..